integration/generator/resources/ec2_linux_test_matrix.json

```diff
@@ -3,7 +3,7 @@
     "os": "ubuntu-20.04",
     "username": "ubuntu",
     "instanceType":"t3a.xlarge",
-    "installAgentCommand": "sudo dpkg -i -E ./amazon-cloudwatch-agent.deb",
+    "installAgentCommand": "go run integration/deb/install_deb.go",
```

Comment:
Would it make more sense to write a bash script and call that rather than wrapping the shell commands inside of a go program?